packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -16,6 +15,8 @@
     display: flex;
     align-items: center;
     font-size: $iui-font-size;
+    min-height: $iui-component-height;
+    height: 100%;
     cursor: pointer;
     @media (prefers-reduced-motion: no-preference) {
```

Comment:
We can remove the `@media (prefers-reduced-motion: no-preference)` here and in other places that aren't effecting motion.

Reply:
You will need to help me here..
You want me to remove all the media queries from this file?

Reply:
Hypothetical example not from `tabs.scss`:
```scss
transform: rotate(90deg);
transition: background-color $iui-speed-fast ease;
@media (prefers-reduced-motion: no-preference) {
  transition: background-color $iui-speed-fast ease, transform $iui-speed-fast ease;
}
```
The above example would allow `background-color` to transition for all users & the media query would allow `background-color` & `transform` (motion) to animate for users who don't enable reduced motion.  For users who don't enable reduced motion, that media query would override the previous line.

Looking through `tabs.scss`, most of the transitions have to do with `background-color` & `fill` which are not motion.  The transitions that target `width` & `height` are motion so those can stay within the media query.

We have been misusing the media query and wrapping pretty much all transitions, but now we know it should only target motion specifically.